src/new-client/src/app/lib/services/appointmentsService.ts

```diff
@@ -235,3 +238,25 @@ export const saveAvailability = async (
   // TODO: Once the fetch availability route is implemented, refresh the tag here
   // revalidateTag(`fetchAvailability`);
 };
+
+export async function fetchInformationForCitizens(site: string, scope: string) {
+  const response = await appointmentsApi.get<SiteWithAttributes>(
+    `sites/${site}?scope=${scope}`,
+  );
+
+  return handleBodyResponse(response)?.attributeValues ?? [];
+}
+
+export const setSiteInformationForCitizen = async (
+  site: string,
+  attributeValues: SetAttributes,
+) => {
+  const response = await appointmentsApi.post(
+    `sites/${site}/attributes`,
+    JSON.stringify(attributeValues),
+  );
+
+  // TODO: Notification?
+  handleBodyResponse(response);
+  revalidatePath(`/site/${site}/information-for-citizens`);
```

Comment:
I might've misread, but I don't think this is the right path? Looks like it should be `/site/${site}/details`? Admittedly, I'm not 100% certain these calls are actually having the intended effect yet so I do intend to investigate all our caching stuff at some point